xstream/src/java/com/thoughtworks/xstream/mapper/Mapper.java

```diff
@@ -47,6 +47,14 @@
      */
     boolean isImmutableValueType(Class<?> type);
 
+    /**
+     * Whether this type is an immutable type whose references must be kept anyways
+     * (for compatibility) at deserialization.
+     *
+     * @since 1.4.9
```

Comment:
not sure what to do about this, do you guys enforce `@since` tags, and whats the correct value?